docs/content/metric-stores/overview.md

```diff
@@ -0,0 +1,106 @@
+configurations.
+
+| Key                 | Required | Default   | Type    | Description                                                  |
+| ------------------- | -------- | --------- | ------- | ------------------------------------------------------------ |
+| type                | Required | -         | String  | The type of the metric store to use.                         |
+| report_interval_sec | Required | -         | Integer | The interval in seconds to report metrics.                   |
+| namespace           | Optional | "default" | String  | The namespace to report metrics to the metric store. Metrics within different namespace will not overwrite each other. |
+
+## Defining metrics
+
+Feathub supports defining metrics at feature's granularity. Below is an example
+of defining a metric for a feature.
+
+```python
+f_total_cost = Feature(
+    name="total_cost",
+    transform=SlidingWindowTransform(
+        expr="cost",
+        agg_func="SUM",
+        group_by_keys=["name"],
+        window_size=timedelta(days=2),
+        step_size=timedelta(days=1),
+    ),
+    metrics=[
+    	Count(
+          window_size=timedelta(days=1),
+      )
+    ],
+)
+```
+
+## Metric reporting format
+
+Metrics reported by metric stores will have the following format by default.
+Some metric stores might override the default format, and please check the
+document of each metric store for details.
+
+- Metric name: `"{namespace}_{feature_name}_{metric_type}"`
+  - namespace: The namespace of the metric store.
+  - feature_name: The name of the host feature.
+  - metric_type: The type of the metric.
+- Metric labels:
+  - namespace: The namespace of the metric store.
+  - table_name: The name of the sink where the host features would be written
+    to.
+  - feature_name: The name of the host feature.
+  - other labels the metric itself has exposed.
+
+## Built-in metrics
+
+Below are Feathub's built-in metrics's class names, their parameters and their
+exposed tags.
+
+For built-in metrics described below, their metric types equal to their class
+names.
+
+### Count
+
+Count is a metric that shows the number of features. It has the following
+parameters:
+
+- filter_expr: Optional. If it is not None, it represents a partial FeatHub
+  expression which evaluates to a boolean value. The partial Feathub expression
+  should be a binary operator whose left child is absent and would be filled in
+  with the host feature name. For example, "IS NULL" will be enriched into
+  "{feature_name} IS NULL". Only features that evaluate this expression into
+  True will be considered when computing the metric.
+- window_size: The time range to compute the metric. It should be zero or a
+  positive time span. If it is zero, the metric will be computed from all
+  features that have been processed since the Feathub job is created.
+
+It exposes the following tags:
+
+- filter_expr: The value of the filter_expr parameter.
```

Comment:
I will a note in `Metric reporting format` section instead of in the section for each metric.

Besides, PrometheusSink has been able to identify the escaped characters and de-escape them. Do you think other metric stores will also have this ability, and finally the escaping behavior would only exist in internal implementation and not exposed to users?

Reply:
Given that Prometheus will show the label value as if it has not been escaped, I agree it is reasonable not to document how we escape the label value.